src/Resources/contao/forms/FormCaptcha.php

```diff
@@ -42,11 +42,11 @@ class FormCaptcha extends \Widget
 	protected $strCaptchaKey;
 
 	/**
-	 * Security questions
+	 * Captcha values
 	 *
 	 * @var string
 	 */
-	protected $strQuestion;
+	protected $arrCaptcha;
```

Comment:
And this could be `protected $arrCaptcha = array()` if we added the more explicit check I have suggested.